test/render/test-cases.js

```diff
@@ -840,6 +856,7 @@ export const TEST_CASES = [
   },
   {
     name: 'column-lnglat',
+    effects: [lightingEffect],
```

Comment:
- Do not modify an existing test case - we need verification that make sure that each feature works independently.
- Missing tests for layers with shadow disabled